src/lib/build.ts

```diff
@@ -92,18 +116,13 @@ export const getBuildOptions = async ({
       ...getFeatureFlagsFromSiteInfo(cachedConfig.siteInfo),
       functionsBundlingManifest: true,
     },
+    // @ts-expect-error(serhalp) -- TODO(serhalp) Upstream the type fixes above into @netlify/build
```

Comment:
This is because even though we've patched the wrong types, it expects us to pass those wrong types back in here